src/utils/parsers/common.ts

```diff
@@ -147,23 +142,41 @@ export abstract class BaseParser {
         )
     }
 
+    private extractUsageInformation(file: usage.File, match: MatchWithRange): MultilineUsageChunk {
+        const lines: usage.LineItem[] = []
+        let index = 0
+        
+        for (const line of file.lines) {
+            index += line.content.length - 1
+            if (index >= match.range.start && index <= match.range.end) {
+                lines.push(line)
+            }
+        }
+
+        return {
+            content: lines.reduce((prev, curr) => prev + curr.content, ''),
```

Comment:
were you thinking that this could be used to display the matched content? should we join with escaped newline characters so that we could display this formatted?

Reply:
yes that was the idea, i can join them with new lines in another pr !